### PR TITLE
[PM-7176] Create appearance settings component (navigational changes)

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Split out the appearance-settings related controls into a separate page according to the UI re-design (navigational changes)

This only contains navigational changes and makes it easier to enable code-ownership
settings.component is under tools-ownership
and
appearance.component is under vault-ownership

The visual changes planned for the UI refresh will be done, once all needed base components from the Component Library are available.

## Code changes

- **Create appearance settings component -** https://github.com/bitwarden/clients/commit/e5702af52ade8e63348a40f54e4555b9ace167c0
- **Add entry in settings to navigate to the appearance settings page -** https://github.com/bitwarden/clients/commit/b13730fed8e04c2c5cc41baec7fb3642567c775e
- **Add transition animation for settings to appearance and back -** https://github.com/bitwarden/clients/commit/28bb9fa6a8b5974b1a72d1ee2b1e4e0da7bad9fc
- **Remove settings from options that are now under appearance -** https://github.com/bitwarden/clients/commit/d74d8449c8fcfe10ed3f2cf5b9d54fdc9c3b3e90

### Before
before_options

### After
after_options
after_settings
after_appearance

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by LRNcardozoWDF — DISMISSED** [review]
Nice job, LGTM!

File: apps/browser/src/vault/popup/settings/appearance.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    private domainSettingsService: DomainSettingsService,
+    private badgeSettingsService: BadgeSettingsServiceAbstraction,
+    i18nService: I18nService,
+    private themeStateService: ThemeStateService,
+  ) {
+    this.themeOptions = [
+      { name: i18nService.t("default"), value: ThemeType.System },
+      { name: i18nService.t("light"), value: ThemeType.Light },
+      { name: i18nService.t("dark"), value: ThemeType.Dark },
+      { name: "Nord", value: ThemeType.Nord },
```
Comment (audrey-jensen) [inline apps/browser/src/vault/popup/settings/appearance.component.ts:35]: ⛏️ Looks like there's a missing i18n lookup here.

File: apps/browser/src/popup/app-routing.module.ts [hunk 2]
```diff
@@ -293,6 +294,12 @@ const routes: Routes = [
     canActivate: [AuthGuard],
     data: { state: "options" },
   },
+  {
+    path: "appearance",
+    component: AppearanceComponent,
+    canActivate: [AuthGuard],
+    data: { state: "appearance" },
```
Comment (audrey-jensen) [inline apps/browser/src/popup/app-routing.module.ts:301]: ⛏️ Should the settings views call themselves out in their paths? Here there wouldn't be confusion. I'm calling it out only for consistency with #8998.

**Review by audrey-jensen — APPROVED** [review]
⛏️ nitpicks

**Review by audrey-jensen — APPROVED** [review]

**Review by LRNcardozoWDF — APPROVED** [review]

**Pull request merged**